### ISSUE TITLE
Can you cause the Frame-Generation feature of FSR 3 to affect the GUI's icons, too?

### ISSUE THREAD

**Issue by ancient-animal** [Lyall/FFXVIFix#8]
The frame-generation feature functions perfectly for me, because it could boost 30 FPS to 60. It's extremely smooth, but the icons that move, like the red quest marker, and the chatboxes above characters, still are rendered with 30 FPS, so the beautifully smooth game is ruined by the stuttering icons while turning the camera. I know FSR3 is specifically designed to NOT affect the GUI, but could you force it to?

**Comment by LeroySinclaire** [comment]
Yes, this would be super nice. The icons on the HUD jittering like that is really irritating.

**Comment by kamcoi** [comment]
up

**Comment by sylveon** [comment]
Another thing you can notice is that the "transparency" effect of the UI uses the native frame as source, not the frame gen frames. This is most noticeable when you have quick travel prompt (e.g. after a sidequest) and move your camera around quickly while the native FPS is low. It will practically look like it's tearing underneat the UI.

I personally decided to combine FSR quality combined with framegen. It bumps the native FPS enough that the UI runs decently smooth. (though i don't like the amount of smearing that FSR quality produces in cutscenes...)
